core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -671,8 +671,9 @@ private[spark] class TaskSetManager(
           if (blacklistedEverywhere) {
             val partition = tasks(indexInTaskSet).partitionId
             abort(s"Aborting $taskSet because task $indexInTaskSet (partition $partition) " +
-              s"cannot run anywhere due to node and executor blacklist.  Blacklisting behavior " +
-              s"can be configured via spark.blacklist.*.")
+              s"cannot run anywhere due to node and executor blacklist." +
```

Comment:
Can you please prettify the output message a bit. From what I saw in the PR description, it is a bit messy there.